### PR TITLE
5.7 cloud provisioning

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================

__Fixing__  Azure Provisioning because 'vm_size' became lowercase in 5.7.0.6 + removing useless sel_force_navigate which started ocasionally failing with Namazing/wigetastic updates

{{pytest: cfme/tests/cloud/test_provisioning.py::test_provision_from_template --use-provider azure --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7d770c454ee288178e14089f89deb24deef0a2c6:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by lkhomenk** [comment]
waiting connectivity issue between PRT and SCVMM's wsman to be resolved

File: cfme/tests/cloud/test_provisioning.py [hunk 1]
```diff
@@ -47,14 +46,12 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
...
         'notes': note,
-        'instance_type': provisioning['instance_type'] if provider.type != "azure" else
-        provisioning['vm_size'],
-        "availability_zone": provisioning["availability_zone"] if provider.type != "azure" else
-        None,
-        'security_groups': [provisioning['security_group']] if provider.type != "azure" else
-        provisioning["network_nsg"],
-        "guest_keypair": provisioning["guest_keypair"] if provider.type != "azure" else None
     }
+    if not isinstance(provider, AzureProvider):
```
Comment (izapolsk) [inline cfme/tests/cloud/test_provisioning.py]: wouldn't it be better to merge this isinstance for Azure with same statement below ?

File: cfme/tests/cloud/test_provisioning.py [hunk 2]
```diff
@@ -47,14 +46,12 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
         'first_name': 'Image',
         'last_name': 'Provisioner',
         'notes': note,
-        'instance_type': provisioning['instance_type'] if provider.type != "azure" else
-        provisioning['vm_size'],
```
Comment (izapolsk) [inline cfme/tests/cloud/test_provisioning.py]: are you sure such options don't need to be present for OpenStackProvider, GCEProvider ?

File: cfme/tests/cloud/test_provisioning.py (hunk 1 above)
Comment (izapolsk) [inline cfme/tests/cloud/test_provisioning.py]: wouldn't it be better to merge this isinstance for Azure with same statement below ?

File: cfme/tests/cloud/test_provisioning.py (hunk 2 above)
Comment (lkhomenk) [inline cfme/tests/cloud/test_provisioning.py]: Azure has a bit another naming/parameters, but these inside "not isinstance" are common for all other providers.Azure gets the same provisioning options inside "isinstance Azure"

Comment (lkhomenk) [inline cfme/tests/cloud/test_provisioning.py]: the other way to do this is to change yamls(make azure parameters common). but that should be done carefully as I'm not fully aware which tests can be affected

File: cfme/tests/cloud/test_provisioning.py (hunk 1 above)
Comment (lkhomenk) [inline cfme/tests/cloud/test_provisioning.py]: before it had a lot of IF-conditions, and not it got one more inside Azure. so now it is more readable

File: cfme/tests/cloud/test_provisioning.py (hunk 2 above)
Comment (izapolsk) [inline cfme/tests/cloud/test_provisioning.py]: my error, I missed "not azure"

File: cfme/tests/cloud/test_provisioning.py (hunk 1 above)
Comment (izapolsk) [inline cfme/tests/cloud/test_provisioning.py]: that's clear. I mistook both if azure statements.

**Comment by lkhomenk** [comment]
I'll add gce fix here - as it started failing in 5.7.0.7 version

**Comment by lkhomenk** [comment]
https://github.com/ManageIQ/wrapanapi/pull/82 this could also help with azure test setups

**Comment by lkhomenk** [comment]
http://10.16.4.32/trackerbot/pr/run/8104 - this one is for gce and ec2west

**Comment by lkhomenk** [comment]
http://10.16.4.32/trackerbot/pr/run/8118 - 3/3 for azure

**Pull request merged**